src/renderer/components/ImageCropper/styles.module.scss

```diff
@@ -0,0 +1,41 @@
+// We need to force the dialog to full height to make `AutoSizer` work
+.imageCropperDialog {
+  height: 100%;
+}
+
+.imageCropperDialogBody {
+  display: flex;
+  flex-direction: column;
+}
```

Comment:
To me it looks like this is not really required. Also there's no use of `AutoSizer` I can spot?